components/vf-hero/vf-hero--example.njk

```diff
@@ -0,0 +1,48 @@
+<style>
+  :root {
+    --vf-hero-bg-image: url('{{ '../../assets/vf-hero/assets/vf-intro-group.png' | path }}');
```

Comment:
Should this be pass from the config yml?

Reply:
Yeah - I initially had issues w/ `.yml` and `--custom-properties` so threw it here.

We need to think about how we can consolidate all things in `:root`, it it's needed, if it's possible, if we can stop duplication. etc.

Reply:
Yeah, have had a similar feeling -- but as of yet no good suggestions 🤔